apps/mobile/lib/features/authentication/test/domain/use_cases/authen_value_objects_test.dart

```diff
@@ -44,6 +45,62 @@ void main() {
       expect(result, left(const AuthFailure.serverError()));
     });
 
-    // Test các phương thức khác tương tự
+    test('logout() should complete successfully', () async {
+      await expectLater(authFacade.logout(), completes);
+    });
+
+    test('changePassword() should change password successfully', () async {
+      final oldPassword = Password('old_password');
+      final newPassword = Password('new_password');
+      final confirmNewPassword = Password('new_password');
+
+      final result = await authFacade.changePassword(
+        oldPassword: oldPassword,
+        newPassword: newPassword,
+        confirmNewPassword: confirmNewPassword,
+      );
+
+      expect(result, isA<Right<AuthFailure, Unit>>());
+    });
+
+    test(
+        'changePassword() should fail with AuthFailure for password dont match',
+        () async {
+      final oldPassword = Password('wrong_old_password');
+      final newPassword = Password('new_password');
+      final confirmNewPassword = Password('new_password');
+
+      final result = await authFacade.changePassword(
+        oldPassword: oldPassword,
+        newPassword: newPassword,
+        confirmNewPassword: confirmNewPassword,
+      );
+
+      expect(result, isA<Left<AuthFailure, Unit>>());
+      expect(result.leftMap((l) => l is PasswordsDontMatch), Left(true));
```

Comment:
Cái kiểu Left + Right nay, t thấy nó không clear code conversion --> Move qua OK or Fail còn rõ ràng hơn về mặt ý nghĩa code.

Reply:
Nhớ check toàn bộ lại phần Left + Right này nhé